README.md

```diff
@@ -766,7 +766,8 @@ myService.getResource({
     const { result, status, headers, statusText } = response;
     console.log(JSON.stringify(headers, null, 4));
   }).catch((err) => {
-    console.log(JSON.stringify(err, null, 4));
+    console.log("Error status code: " + err.status + " (" + err.statusText + ")");
```

Comment:
Pretty much all errors will have a `message` property but not all will necessarily have `status` and `statusText` properties - only errors that occur because of error responses. For example, if the request never makes a connection to the service, because of certificate problems, I don't think the error object will have those fields.

I'm fine with this being here but to be as "type safe" as we can in JS, we should wrap this line in an if statement:
```suggestion
    if (err.status && err.statusText) {
      console.log("Error status code: " + err.status + " (" + err.statusText + ")");
    }
```